google/cloud/aiplatform/jobs.py

```diff
@@ -534,6 +536,13 @@ def create(
                 be immediately returned and synced when the Future has completed.
             create_request_timeout (float):
                 Optional. The timeout for the create request in seconds.
+            batch_size (Optional[int]):
```

Comment:
Please move the Optional type hint down into the description. Similar to the argument above.